src/components/Table/Table.tsx

```diff
@@ -149,8 +149,6 @@ export const TableCell = styled(
     ),
 )`
     display: inline-flex;
-    align-items: baseline;
-    flex-wrap: wrap;
```

Comment:
Maybe overrride from props instead of remove css properties?

Reply:
These styles are not actually used anywhere, so I think they are redundant